web/src/types.ts

```diff
@@ -12,4 +12,5 @@ export type TPackage = {
   name: string
   description: string
   components: TComponentRatios
+  molecularWeight: number
```

Comment:
Isn't molecular weight a constant of the chemical compound? If so, I don't think it makes sense to store it as part of the package

Reply:
Ah, no, okay, I get it. Could we perhaps call it something else then? combinedMolecularWeight, molecularWeightSum or something like that? I see you call it feedMolecularWeight in FeedFlowInput, but I'm not sure that is better. Best if it's called the same everywhere, though

Reply:
I agree. Will rename it.